src/unread/__tests__/unreadModel-test.js

```diff
@@ -288,23 +304,45 @@ describe('stream substate', () => {
       // prettier-ignore
       expect(summary(state)).toEqual(Immutable.Map([
         [123, Immutable.Map([['foo', [1, 2, 3]], ['qux', [7]]])],
-        [234, Immutable.Map([['bar', [4, 5]]])],
+        [234, Immutable.Map([['bar', [4, 5, 100]]])],
       ]));
 
       const action = mkAction({ messages: [1, 2] });
       const newState = reducer(state, action, eg.plusReduxState);
       // prettier-ignore
       expect(summary(newState)).toEqual(Immutable.Map([
         [123, Immutable.Map([['foo', [3]], ['qux', [7]]])],
-        [234, Immutable.Map([['bar', [4, 5]]])],
+        [234, Immutable.Map([['bar', [4, 5, 100]]])],
```

Comment:
We can add the extra message locally for just the one test case where it's being used, to avoid adding noise to the other test cases like this. Same way that the "when removing, don't touch unaffected topics or streams" case already adds a message of its own.